.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: java
-jdk: oraclejdk7
+jdk: oraclejdk8
```

Comment:
I tried to use Java 7 and downgrade checkstyle version but, it doesn't work for now. The fix might require several hack.

I think that we can use Java 8 on Travis CI as we don't use Java 8 for building and releasing the plugin. I added comment why we need oraclejdk8 on Travis CI. 3d90298